source/upgrade.txt

```diff
@@ -59,11 +59,6 @@ Version 4.2 Breaking Changes
   to use a new format that minimizes serialization compatibility issues
   across different versions of the driver.
 
-  Any objects that include an ``ObjectId`` field that were serialized with an
```

Comment:
I think we should say something here, but just make it clear that it's Java Object Serialization that we are referring to and not BSON serialization.  Something like

Any Java object containing an ``Objectid` that have been serialized via Java Object Serialization[1] with an older version of the driver .... 

Note that this does _not_ effect ObjectId instances that have been serialized to BSON and stored in MongoDB.

[1] https://docs.oracle.com/javase/tutorial/jndi/objects/serial.html